### ISSUE TITLE
chromium v22 cannot read CSS in #hack-448c4b5ec6e62bb3c328cca118f0a80d

### ISSUE THREAD

**Issue by gamifiq** [4ae9b8/browserhacks#171]
Testing hack http://browserhacks.com/#hack-448c4b5ec6e62bb3c328cca118f0a80d
on my raspberry's mot recent Chromium version : 22.0.1229.94 built on Debian 7.0 running on debian 7.2 (161065)
I was surprised to see that CSS ignored by chromium :  
 
body {
        opacity: 0; /*read by chromium */
    }
 @media screen {
        @media (min-width: 0px) {
            body {
                opacity: 1; /*NOT read by chromium  v22*/
            }
        }
    }

This hack is supposed to be for "Everything but Internet Explorer and Safari ≤6".
However, using Chromium (Version 37.0.2062.120 Ubuntu 12.04) that same CSS was actually interpreted.

I suggest you may add a warning/info about it

**Comment by jeffclayton** [comment]
You are correct but not an 'everything but Safari' technically, but older versions are supported - the nested media query was introduced into Chrome/Chromium in version 26. Also, Firefox did not recognize it until version 11 though very few people use that old of a version of Firefox to know this. Safari 6.1+, Chrome 26+, Firefox 11+

Since . MSIE did not support it either (11-) Internet Explorer is not part of that list either, bit the new MS Edge Browser, which starts where MSIE left off at version 12, does (Edge 12+)  -- so add Edge 12+ to the above list for this actual set of stats: 

Safari 6.1+, Chrome 26+, Firefox 11+, Edge 12+
